lib/modules/txt.mjs

```diff
@@ -314,3 +314,29 @@ export async function validateCopyrightSection (doc, { mode = MODES.NORMAL } = {
 
   return result
 }
+
+/**
+ * Validate copyright date
+ *
+ * @param {Object} doc Document to validate
+ * @param {Object} [opts] Additional options
+ * @param {number} [opts.mode=0] Validation mode to use
+ * @param {number} opts.year Expect the given year in the boilerplate
+ * @returns {Array} List of errors/warnings/comments or empty if fully valid
+ */
+export async function validateCopyrightDate (doc, { mode = MODES.NORMAL, year } = {}) {
+  const result = []
+
+  const yearToCheck = year || new Date().getFullYear()
```

Comment:
There is a problem, that you compare string to number. From text you get string year, but from new Date().getFullYear() you get number. Please use one unified type. Also we get number from cli parametes